tssc/step_implementers/package/maven.py

```diff
@@ -176,6 +180,22 @@ def required_runtime_step_config_keys():
         """
         return REQUIRED_CONFIG_KEYS
 
+    def _settings_file(self):
```

Comment:
@ppremru why not call this `__generate_maven_settings`?

Reply:
Agreed. Don't forget to make the same change in all other maven-based step implementers.